modules/dbparser/tests/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 add_unit_test(TARGET test_timer_wheel DEPENDS patterndb)
 add_unit_test(TARGET test_patternize DEPENDS patterndb syslogformat)
-add_unit_test(CRITERION LIBTEST TARGET test_patterndb DEPENDS patterndb basicfuncs syslogformat)
+add_unit_test(LIBTEST TARGET test_patterndb DEPENDS patterndb basicfuncs syslogformat)
```

Comment:
This patch reverts this line, this line should not be changed in this patch.

Reply:
The 2nd patch modified the `add_unit_test(... test_patterndb ...)`, so it is `add_unit_test(CRITERION LIBTEST TARGET test_patterndb DEPENDS patterndb basicfuncs syslogformat)` with Criterion.
And the last (3rd patch), reverts the above change, so the result is the `test_patterndb` is marked as non-criterion test.
(In the unified diff it cannot be seen.)

Reply:
Thanks for the finding. I clean up the commits.